sbin/build.sh

```diff
@@ -253,6 +253,137 @@ patchFreetypeWindows() {
   fi
 }
 
+# Returns the version numbers in version-numbers.conf as a space-seperated list of integers.
+# e.g. 17 0 1 5
+# or 8 0 0 432
+# Build number will not be included, as that is not stored in this file.
+versionNumbersFileParser() {
+  funcName="versionNumbersFileParser"
+  # Find version-numbers.conf or the equivalent and confirm we can read it.
+  buildSrc="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}"
+  # File location for jdk8.
+  numbersFile="${buildSrc}/common/autoconf/version-numbers"
+  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -eq 11 ]; then
+    # File location for JDK11.
+    numbersFile="${buildSrc}/make/autoconf/version-numbers"
+  elif [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 17 ]; then
+    # File location for JDK17 and up.
+    numbersFile="${buildSrc}/make/conf/version-numbers.conf"
+  fi
+
+  # Check the version numbers file exists.
+  if [ ! -r "${numbersFile}" ]; then
+    echo "ERROR: build.sh: ${funcName}: The file expected to contain the JDK version numbers could not be found and/or read." >&2
+    echo "ERROR: build.sh: ${funcName}: The expected location was: ${numbersFile}" >&2
+    exit 1
+  fi
+
+  error="false"
+
+  fileVersionArray=()
```

Comment:
Could this be done with a single more efficient `awk` command to do a single pass on the file e.g.
`awk -F= '/^JDK_MAJOR_VERSION=/{MAJOR=$2} /^JDK_MINOR_VERSION=/{MINOR=$2} END{print MAJOR " " MINOR}' common/autoconf/version-numbers`
which would give `1 8` for jdk8u for example? The `END` part of the awk can spit it out in whatever format you want  avoiding the need for the "arrange the file version array into a standard format for that feature number" version later on.